frame-metadata/src/lib.rs

```diff
@@ -72,14 +72,14 @@ pub mod v13;
 pub mod v14;
 
 /// Metadata v15
-#[cfg(feature = "unstable")]
+#[cfg(feature = "current")]
 pub mod v15;
 
 // Reexport all the types from the latest version.
 //
 // When a new version becomes available, update this.
 #[cfg(feature = "current")]
-pub use self::v14::*;
+pub use self::v15::*;
```

Comment:
Sortof unrelated, but to me it feels wrong to re-export all of the latest types like this; people will encounter fewer breaking changes if they have to explicitly ask for `v14::Foo` or `v15::Foo` so I'd rather that they have to be explicit :)

Reply:
@ascjones perhaps you have an opinion on this!

Reply:
Yep I agree :D We should ditch this `pub use self::v[num]::*` reexport.

Let's wait for @ascjones to confirm this! Thanks 🙏

Reply:
Fine by me :+1: